### PR TITLE
Add description and examples for SQL Keywords

### PR DESCRIPTION
**Pull request by estherk15**
### What does this PR do? What is the motivation?

- Add examples and descriptions for SQL keyword comparators

### Merge instructions

Pending PM review

### CODE REVIEW

File: content/en/ddsql_editor/reference/expressions_and_operators.md [hunk 1]
```diff
@@ -39,20 +39,27 @@ DDSQL implements the following comparison operators:
 
 For tag references and tag groups, the equality operator (`=`) is treated as a "contains" comparison. See the [Querying Tags in DDSQL][1] for more details.
 
-Additionally, DDSQL supports the following SQL keywords, which function as standard boolean operators:
+## SQL comparison keywords
 
-- `NOT`
-- `AND`
-- `OR`
+DDSQL supports the following SQL keywords, which function as standard boolean operators:
```
Comment (estherk15) [inline content/en/ddsql_editor/reference/expressions_and_operators.md:44]:
```suggestion
DDSQL supports the following SQL keywords, which function as standard Boolean operators:
```

File: content/en/ddsql_editor/reference/expressions_and_operators.md [hunk 2]
```diff
@@ -39,20 +39,27 @@ DDSQL implements the following comparison operators:
...
+## SQL comparison keywords
 
-- `NOT`
-- `AND`
-- `OR`
+DDSQL supports the following SQL keywords, which function as standard Boolean operators:
+
+| Operator | Description            | Example | Result |
+|----------|------------------------|---------|--------|
+| `NOT`    | Filter records based on more than one condition. | `SELECT * FROM Student WHERE NOT Major = 'Economics';`   | Return all students that do not major in Economics.  |
```
Comment (joparisot) [inline content/en/ddsql_editor/reference/expressions_and_operators.md:48]: Can we generally replace these examples by examples relevant in the context of Datadog? I'll give you a few
Here, use 
select * from host where not env = 'prod'

File: content/en/ddsql_editor/reference/expressions_and_operators.md [hunk 3]
```diff
@@ -39,20 +39,27 @@ DDSQL implements the following comparison operators:
...
+| `AND`    | Filter records based on more than one condition. | `SELECT * FROM Student WHERE Age = 18 AND Major = 'Economics';`   | Return all students that are age 18 AND major in Economics.  |
```
Comment (joparisot) [inline content/en/ddsql_editor/reference/expressions_and_operators.md:49]: select * from host where env = 'prod'and cloud_provider = 'aws'

File: content/en/ddsql_editor/reference/expressions_and_operators.md [hunk 4]
```diff
@@ -39,20 +39,27 @@ DDSQL implements the following comparison operators:
...
+| `OR`     | Filter records based on more than one condition. | `SELECT * FROM Student WHERE Age = 18 OR Major = 'Economics';`   | Return all students that are either age 18 OR major in Economics.  |
```
Comment (joparisot) [inline content/en/ddsql_editor/reference/expressions_and_operators.md]: select * from host where env = 'prod' or cloud_provider = 'aws'

File: content/en/ddsql_editor/reference/expressions_and_operators.md [hunk 5]
```diff
@@ -39,20 +39,27 @@ DDSQL implements the following comparison operators:
...
 
-- `IS NULL`
-- `IS NOT NULL`
-- `LIKE`
-- `NOT LIKE`
-- `IN`
-- `NOT IN`
+| Operator     | Description            | Example | Result |
+|--------------|------------------------|---------|--------|
+| `IS NULL`    | Select rows if the specified field is null. | `SELECT * FROM albums WHERE artist IS NULL`   | Return all rows that contain no data in the `artist` column.  |
```
Comment (joparisot) [inline content/en/ddsql_editor/reference/expressions_and_operators.md:56]: select *
from host
where cloud_provider IS NULL

File: content/en/ddsql_editor/reference/expressions_and_operators.md [hunk 6]
```diff
@@ -39,20 +39,27 @@ DDSQL implements the following comparison operators:
...
+| `IS NOT NULL`| Select rows if the specified field is not null. Exclude rows with missing data. | SELECT * FROM albums WHERE artist IS NOT NULL | Return all rows that contain data in the `artist` column.   |
```
Comment (joparisot) [inline content/en/ddsql_editor/reference/expressions_and_operators.md:57]: select *
from host
where cloud_provider IS NOT NULL

File: content/en/ddsql_editor/reference/expressions_and_operators.md [hunk 7]
```diff
@@ -39,20 +39,27 @@ DDSQL implements the following comparison operators:
...
+| `LIKE`       | Search for a specific pattern in a string value. You can use the following wildcard characters to define the patterns: <br>**Percent sign (%)**: Represents zero, one, or multiple characters. <br>**Underscore (_)**: Represents a single character. | `SELECT * FROM employees WHERE firstname LIKE 'a%';` | Return all rows from the `employees` table where the `firstname` column value starts with `a`.  |
+| `NOT LIKE`   | Exclude rows from a search, where the row has a specific pattern in a string value. You can use the wildcards `%` and `_` for pattern matching. | `SELECT * FROM products WHERE product_code NOT LIKE '_B%';` | Return all rows from the `products` table where the `product_code` does **not** have `B` as the second character. |
```
Comment (joparisot) [inline content/en/ddsql_editor/reference/expressions_and_operators.md:59]: SELECT *
FROM aws_eks_cluster
WHERE LOWER(logging) NOT LIKE '%"enabled":true%'

File: content/en/ddsql_editor/reference/expressions_and_operators.md [hunk 8]
```diff
@@ -39,20 +39,27 @@ DDSQL implements the following comparison operators:
...
+| `IN`         | Find multiple values in a `WHERE` clause. The `IN` operator is shorthand for multiple `OR` conditions. | `SELECT * FROM Customers WHERE Country IN ('Germany', 'France', 'UK');`  | Return all rows from `Customers` table where the `Country` value is either 'Germany', 'France', or 'UK'.|
```
Comment (joparisot) [inline content/en/ddsql_editor/reference/expressions_and_operators.md:60]: select *
from host
where cloud_provider IN ('aws', 'gcp')

File: content/en/ddsql_editor/reference/expressions_and_operators.md [hunk 9]
```diff
@@ -39,20 +39,27 @@ DDSQL implements the following comparison operators:
...
+| `NOT IN`     | Replace a set of arguments with the `<>` or `!=` operator that is combined with the `AND` operator| `SELECT Name FROM Emp WHERE age NOT IN (23, 22, 21);`  | Return the `Names` of the `Employees` who do not have `Age` equal to `23`, `22`, or `21`. |
```
Comment (joparisot) [inline content/en/ddsql_editor/reference/expressions_and_operators.md:61]: select *
from host
where cloud_provider NOT IN ('aws', 'gcp')

File: content/en/ddsql_editor/reference/expressions_and_operators.md (hunk 5 above)
Comment (estherk15) [reply content/en/ddsql_editor/reference/expressions_and_operators.md:56]:
```suggestion
| `IS NULL`    | Select rows if the specified field is null. | `SELECT * FROM host WHERE cloud_provider IS NULL`   | Return all rows that contain no data in the `cloud_provider` column.  |
```

File: content/en/ddsql_editor/reference/expressions_and_operators.md (hunk 6 above)
Comment (estherk15) [reply content/en/ddsql_editor/reference/expressions_and_operators.md:57]:
```suggestion
| `IS NOT NULL`| Select rows if the specified field is not null. Exclude rows with missing data. | SELECT * FROM host WHERE cloud_provider IS NOT NULL | Return all rows that contain data in the `cloud_provider` column.   |
```

File: content/en/ddsql_editor/reference/expressions_and_operators.md [hunk 10]
```diff
@@ -39,20 +39,27 @@ DDSQL implements the following comparison operators:
...
-- `IS NOT NULL`
-- `LIKE`
-- `NOT LIKE`
-- `IN`
-- `NOT IN`
+| Operator     | Description            | Example | Result |
+|--------------|------------------------|---------|--------|
+| `IS NULL`    | Select rows if the specified field is null. | `SELECT * FROM albums WHERE artist IS NULL`   | Return all rows that contain no data in the `artist` column.  |
+| `IS NOT NULL`| Select rows if the specified field is not null. Exclude rows with missing data. | SELECT * FROM albums WHERE artist IS NOT NULL | Return all rows that contain data in the `artist` column.   |
+| `LIKE`       | Search for a specific pattern in a string value. You can use the following wildcard characters to define the patterns: <br>**Percent sign (%)**: Represents zero, one, or multiple characters. <br>**Underscore (_)**: Represents a single character. | `SELECT * FROM employees WHERE firstname LIKE 'a%';` | Return all rows from the `employees` table where the `firstname` column value starts with `a`.  |
```
[the remark this answers is not in the document]
Comment (estherk15) [reply content/en/ddsql_editor/reference/expressions_and_operators.md:58]:
```suggestion
| `LIKE`       | Search for a specific pattern in a string value. You can use the following wildcard characters to define the patterns: <br>**Percent sign (%)**: Represents zero, one, or multiple characters. <br>**Underscore (_)**: Represents a single character. | `SELECT * FROM aws_eks_cluster WHERE LOWER(logging) LIKE '%"enabled":true%' | Return all rows from the `aws_eks_cluster` table where the `logging` column is `"enabled":true`.  |
```

File: content/en/ddsql_editor/reference/expressions_and_operators.md (hunk 7 above)
Comment (estherk15) [reply content/en/ddsql_editor/reference/expressions_and_operators.md:59]:
```suggestion
| `NOT LIKE`   | Exclude rows from a search, where the row has a specific pattern in a string value. You can use the wildcards `%` and `_` for pattern matching. | `SELECT * FROM aws_eks_cluster WHERE WHERE LOWER(logging) NOT LIKE '%"enabled":true%'` | Return all rows from the `aws_eks_cluster` table where the `logging` does **not** have `"enabled":true%'`. |
```

File: content/en/ddsql_editor/reference/expressions_and_operators.md (hunk 8 above)
Comment (estherk15) [reply content/en/ddsql_editor/reference/expressions_and_operators.md:60]:
```suggestion
| `IN`         | Find multiple values in a `WHERE` clause. The `IN` operator is shorthand for multiple `OR` conditions. | `SELECT * FROM host WHERE cloud_provider IN ('aws', 'gcp')`  | Return all rows from `host` table where the `cloud_provider` value is either 'aws', or 'gcp'.|
```

File: content/en/ddsql_editor/reference/expressions_and_operators.md (hunk 9 above)
Comment (estherk15) [reply content/en/ddsql_editor/reference/expressions_and_operators.md:61]:
```suggestion
| `NOT IN`     | Replace a set of arguments with the `<>` or `!=` operator that is combined with the `AND` operator| `SELECT * FROM host WHERE cloud_provider NOT IN ('aws', 'gcp');`  | Return all rows where `cloud_provider` is not `aws` or `gcp`. |
```

File: content/en/ddsql_editor/reference/expressions_and_operators.md [hunk 11]
```diff
@@ -39,20 +39,27 @@ DDSQL implements the following comparison operators:
...
-- `IS NULL`
-- `IS NOT NULL`
-- `LIKE`
-- `NOT LIKE`
-- `IN`
-- `NOT IN`
+| Operator     | Description            | Example | Result |
+|--------------|------------------------|---------|--------|
+| `IS NULL`    | Select rows if the specified field is null. | `SELECT * FROM host WHERE cloud_provider IS NULL`   | Return all rows that contain no data in the `cloud_provider` column.  |
+| `IS NOT NULL`| Select rows if the specified field is not null. Exclude rows with missing data. | SELECT * FROM host WHERE cloud_provider IS NOT NULL | Return all rows that contain data in the `cloud_provider` column.   |
```
Comment (estherk15) [inline content/en/ddsql_editor/reference/expressions_and_operators.md:57]:
```suggestion
| `IS NOT NULL`| Select rows if the specified field is not null. Exclude rows with missing data. | `SELECT * FROM host WHERE cloud_provider IS NOT NULL;` | Return all rows that contain data in the `cloud_provider` column.   |
```

File: content/en/ddsql_editor/reference/expressions_and_operators.md [hunk 12]
```diff
@@ -39,20 +39,27 @@ DDSQL implements the following comparison operators:
...
 
-- `IS NULL`
-- `IS NOT NULL`
-- `LIKE`
-- `NOT LIKE`
-- `IN`
-- `NOT IN`
+| Operator     | Description            | Example | Result |
+|--------------|------------------------|---------|--------|
+| `IS NULL`    | Select rows if the specified field is null. | `SELECT * FROM host WHERE cloud_provider IS NULL`   | Return all rows that contain no data in the `cloud_provider` column.  |
```
Comment (estherk15) [inline content/en/ddsql_editor/reference/expressions_and_operators.md:56]:
```suggestion
| `IS NULL`    | Select rows if the specified field is null. | `SELECT * FROM host WHERE cloud_provider IS NULL;`   | Return all rows that contain no data in the `cloud_provider` column.  |
```

**Review by maycmlee — APPROVED** [review]

**Pull request merged**